src/components/Owners/Providerscard.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types'; 
+// import haircutImg from "../Images/haircut.svg";
+import maleServiceProvider from "../Images/maleServiceprovider.png";
+import femaleServiceProvider from "../Images/femaleServiceproviders.jpg";
+
+
+function Providerscard({providersProperties}) {
```

Comment:
the function name should be camel case providersCard

Reply:
in React functional component must start with capital letter otherwise it gives error

Reply:
Sounds good , you can ignore my comment